BotRoyaleUnity/Assets/TextMesh Pro/Examples & Extras.meta

```diff
@@ -1,8 +1,8 @@
 fileFormatVersion: 2
```

Comment:
Hope you don't mind, I deleted these unused meta files and push it to this branch